csharp/src/Drivers/Databricks/CloudFetch/CloudFetchResultFetcher.cs

```diff
@@ -160,8 +161,11 @@ public async Task StopAsync()
 
                 request.StartRowOffset = offset;
 
+                // Cancelling mid-request breaks the client; Dispose() should not break the underlying client
+                CancellationToken expiringToken = ApacheUtility.GetCancellationToken(DatabricksConstants.DefaultCloudFetchRequestTimeoutSeconds, ApacheUtility.TimeUnit.Seconds);
```

Comment:
should you respect the connection parameter `DatabricksParameters.CloudFetchTimeoutMinutes` instead of the default value?